api/desecapi/pdns_change_tracker.py

```diff
@@ -72,67 +71,15 @@ def api_do(self):
         def pch_do(self):
             raise NotImplementedError()
 
-        def update_catalog(self, delete=False):
-            content = pdns._pdns_patch(
-                pdns.NSMASTER,
-                "/zones/" + pdns.pdns_id(settings.CATALOG_ZONE),
-                {
-                    "rrsets": [
-                        pdns.construct_catalog_rrset(zone=self.domain_name, delete=delete)
-                    ]
-                },
-            )
-            metrics.get("desecapi_pdns_catalog_updated").inc()
-            return content
-
     class CreateDomain(PDNSChange):
         @property
         def axfr_required(self):
             return True
 
         def pdns_do(self):
-            pdns._pdns_post(
-                pdns.NSLORD,
-                "/zones?rrsets=false",
-                {
-                    "name": self.domain_name_normalized,
-                    "kind": "MASTER",
-                    "dnssec": True,
-                    "nsec3param": "1 0 0 -",
-                    "nameservers": settings.DEFAULT_NS,
-                    "rrsets": [
-                        {
-                            "name": self.domain_name_normalized,
-                            "type": "SOA",
-                            # SOA RRset TTL: 300 (used as TTL for negative replies including NSEC3 records)
-                            "ttl": 300,
-                            "records": [
-                                {
-                                    # SOA refresh: 1 day (only needed for nslord --> nsmaster replication after RRSIG rotation)
-                                    # SOA retry = 1h
-                                    # SOA expire: 4 weeks (all signatures will have expired anyways)
-                                    # SOA minimum: 3600 (for CDS, CDNSKEY, DNSKEY, NSEC3PARAM)
-                                    "content": "get.desec.io. get.desec.io. 1 86400 3600 2419200 3600",
-                                    "disabled": False,
-                                }
-                            ],
-                        }
-                    ],
-                },
-            )
-
-            pdns._pdns_post(
-                pdns.NSMASTER,
-                "/zones?rrsets=false",
-                {
-                    "name": self.domain_name_normalized,
```

Comment:
The `domain_name_normalized` property is now obsolete and can be removed

Reply:
removed

Reply:
removed